frontend/components/Main/MainHeader/components.tsx

```diff
@@ -0,0 +1,86 @@
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Button, Flex, Popover, PopoverProps, Typography } from 'antd';
+
+import { COLOR, SUPPORT_URL } from '@/constants';
+import { UNICODE_SYMBOLS } from '@/constants/unicode';
+
+const { Paragraph, Text } = Typography;
+
+const CannotStartAgentText = () => (
+  <Text style={{ color: COLOR.RED }}>
+    Cannot start agent&nbsp;
+    <InfoCircleOutlined />
+  </Text>
+);
+
+export const otherPopoverProps: PopoverProps = {
+  arrow: false,
+  placement: 'bottomRight',
+};
+
+export const JoinOlasCommunity = () => (
+  <div style={{ maxWidth: 340 }}>
+    <Paragraph>
+      Join the Olas community Discord server to report or stay up to date on the
+      issue.
+    </Paragraph>
+
+    <a href={SUPPORT_URL} target="_blank" rel="noreferrer">
+      Olas community Discord server {UNICODE_SYMBOLS.EXTERNAL_LINK}
+    </a>
+  </div>
+);
+
+export const NoRewardsAvailablePopover = () => (
+  <Popover
+    {...otherPopoverProps}
+    content={<JoinOlasCommunity />}
+    title="No rewards available"
+  >
+    <CannotStartAgentText />
+  </Popover>
+);
+
+export const NoJobsAvailablePopover = () => (
```

Comment:
wonder if we could just have one component something like "JoinOlasCommunityPopover" with this content of `JoinOlasCommunity` inside and then we just pass the title prop where it's used, instead of having three different components `NoRewardsAvailablePopover`, `NoJobsAvailablePopover` and `JoinOlasCommunity`.. reduced the code. but up to you 🙏